internal/controller/falcon_container/falconcontainer_controller.go

```diff
@@ -96,6 +96,24 @@ func (r *FalconContainerReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		}
 	}
 
+	validate, err := k8sutils.CheckRunningPodLabels(r.Client, ctx, falconContainer.Spec.InstallNamespace, common.CRLabels("deployment", injectorName, common.FalconSidecarSensor))
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+	if !validate {
+		err = r.StatusUpdate(ctx, req, log, falconContainer,
+			falconv1alpha1.ConditionFailed,
+			metav1.ConditionFalse,
+			falconv1alpha1.ReasonReqNotMet,
+			"FalconContainer must not be installed in a namespace with other workloads running. Please change the namespace in the CR configuration.",
+		)
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+		log.Error(nil, "FalconContainer is attempting to install in a namespace with existing pods. Please update the CR configuration to a namespace that does not have workoads already running.", "namespace", falconContainer.Spec.InstallNamespace, "labels", common.CRLabels("deployment", falconContainer.Name, common.FalconSidecarSensor))
+		return ctrl.Result{}, err
```

Comment:
is it correct that nil error returned here?

Reply:
Don't have preference.
`err` will be `nil` here because of line 110 but code looks as if the intention of programmer is to return an non-nil error 
A comment here to clarify intention would help

Reply:
I'll change it to nil for clarity.